### PR TITLE
Refacto UI attribute mapping

### PR DESCRIPTION
**Pull request by elisegriset92**
From
Capture d’écran 2022-05-02 à 15 36 32

To
Capture d’écran 2022-05-03 à 15 40 30
Capture d’écran 2022-05-03 à 15 40 44

### CODE REVIEW

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 1]
```diff
@@ -46,26 +46,68 @@
...
+                class="rounded-circle mb-1"
+                src="@/assets/images/google-icon-grey.svg"
+                width="20"
+                height="20"
+              >
+              <p>
+                {{ $t('productFeedSettings.summary.tableHeader1') }}
+              </p>
+            </div>
+            <div :style="cssProps" />
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:65]: If all you're doing is having a background image, why not adding the background image to the parent ? l53.

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 2]
```diff
@@ -46,26 +46,68 @@
             class="spinner"
           />
         </div>
-        <b-form-row
-          class="mt-2"
+        <b-container
           v-else
+          class=" maxw-sm-800"
         >
+          <div class="d-flex justify-content-around">
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:53]: It won't be responsive, is it an issue ?

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 3]
```diff
@@ -46,26 +46,68 @@
...
           <template
             v-for="group in attributesToMap"
           >
-            <b-col
+            <div
               v-for="field in group.fields"
               :key="field.label + group.category"
-              :cols="12"
-              :md="6"
+              class="d-flex col-12 "
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:84]: Any reason you don't do ?
```suggestion
              class="row"
```

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 4]
```diff
@@ -46,26 +46,68 @@
...
             >
-              <attribute-field
-                :field="field"
-                :category="group.category"
-              />
-            </b-col>
+              <div class="d-flex col-6">
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:86]: Not responsive either.

If you go with a `.row` l84, you could have:
```suggestion
              <div class="d-flex col-12 col-md-6">
```

And adding the same col class l102 would do the trick.

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 5]
```diff
@@ -139,6 +182,12 @@ export default {
   data() {
     return {
       loading: false,
+      cssProps: {
```
Comment (amaury-hanser) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:185]: Any reason you don't do it in sass instead ?

**Review by amaury-hanser — COMMENTED** [review]
Nice work.  
Few remarks to discussed about, let me know if you find them relevant.

**Review by Quetzacoalt91 — DISMISSED** [review]

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue [hunk 6]
```diff
@@ -46,26 +46,68 @@
             class="spinner"
           />
         </div>
-        <b-form-row
-          class="mt-2"
+        <b-container
           v-else
+          class=" maxw-sm-800"
         >
+          <div class="d-flex justify-content-around flex-md-row flex-column ps-ps-header-attribute-mapping">
```
Comment (Quetzacoalt91) [inline _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue:53]: Some linter feedback to fix

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue (hunk 2 above)
Comment (elisegriset92) [reply _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue]: fixed as seen with product designer

File: _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue (hunk 5 above)
Comment (elisegriset92) [reply _dev/src/components/product-feed/settings/attribute-mapping/attribute-mapping.vue]: Places in a separate scss file

**Review by amaury-hanser — APPROVED** [review]

**Pull request merged**